sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -180,7 +177,18 @@ public ConfigurationAsyncClient buildAsyncClient() {
         policies.add(new ConfigurationCredentialsPolicy(buildCredential));
         HttpPolicyProviders.addBeforeRetryPolicies(policies);
 
-        policies.add(retryPolicy == null ? DEFAULT_RETRY_POLICY : retryPolicy);
+        RetryPolicy defaultRetryPolicy = new RetryPolicy(new ExponentialBackoff() {
+            @Override
+            public Duration calculateRetryDelay(HttpResponse httpResponse, int retryAttempts) {
+                String delay = httpResponse.getHeaderValue(RETRY_AFTER_MS_HEADER);
```

Comment:
Looks like by default we handle following headers in `RetryStrategy` default method:

```
retry-after-ms
 x-ms-retry-after-ms
```

so I guess we don't need this customization in AzConfig?

Reply:
based on what we spoke today, we will only use one response header to avoid any complexity in core. Header name x-ms-retry-after-ms